### ISSUE TITLE
Add class for APRV (Airway pressure release ventilation)

### ISSUE THREAD

**Issue by kshefchek** [monarch-initiative/MAxO#160]
Currently APRV is a synonym of CPAP, but I think this is different enough to be given its own class, see https://en.wikipedia.org/wiki/Airway_pressure_release_ventilation

It's sometimes described as a type of CPAP, or CPAP with release, but I find this definition confusing because often Plow is set to 0 (https://emcrit.org/squirt/aprv/ https://emcrit.org/ibcc/covid19/#invasive_mechanical_ventilation)

BiPAP is also listed as an exact synonym for CPAP, but I would argue it should be narrow or related.

**Comment by LCCarmody** [comment]
Thanks for the input. I'll have to look into it further: Most of those terms were taken from MeSH concepts in an answer to an earlier ticket.  Good to have someone review these.

**Comment by kshefchek** [comment]
Let me know if I can help out with any ventilation related terms.  I'm a little rusty and not at all clinical, but I used to work for a physician who pioneered the use of APRV for patients with ARDS so this topic is near and dear.

**Comment by LCCarmody** [comment]
Okay, I've been reviewing a bit. Nearly all rare disease recommends CPAP or BiPAP, but with the expansion of terms for COVID19, APRV does seem to be a necessary term.  And looking at this, it should be a child term of CPAP. Would you agree with that?

Also, glad you caught that BiPAP is listed as a synonym to CPAP...since MAXO_0000511 is a separate term for BiPAP.  (*there aren't the same stringent checks that HPO has*) .
Also, BiPAP should have also been listed as positive-pressure respiration, but was only listed as noninvasive ventilation.  So...plenty of mistakes!  I'll correct this and merge these to the master soon.  Maybe you could check a bit closer when I'm done.  Thanks for this catch!

**Comment by kshefchek** [comment]
Okay I'm dusting off the cobwebs :) it might be useful to get an ICU respiratory therapist or physician to weigh in here. I'll share what I remember and some of this is likely incorrect:

Classifying modes of ventilation as invasive or non-invasive will break down at some point.  For example, CPAP is commonly non-invasive but is used on patients prior to extubation - I think these are called weaning trials.  Determining if ventilation is invasive or not could be logically inferred (eg if a patient or patient population in a study are intubated).

There are also modes of ventilation and protocols of those modes.  Broadly, there are two parent modes of ventilation: volume controlled and pressure controlled.  There are also other ventilation settings that can impact outcomes, such as PEEP (Positive End Expiratory Pressure) and ATC (automatic tube compensation). And then various protocols around these modes of ventilation that we'll want to capture.  And even within protocols there are differences depending on the patient (eg ARDS patients with COPD are treated differently).

For example, APRV is a type of pressure controlled ventilation that has evolved to have different protocols.  The inventor, John Downs proposed a protocol.  There is another common protocol informally called Habashi APRV.  Then there are other protocols in journals that have led to confusion around the method. Similarly, there are different protocols for volume controlled ventilation (eg ARDSNet is a low volume ventilation protocol for ARDS patients).

I'm unsure how to best represent this all in MAxO, but it's possible we could consider a protocol the medical action and have these be logically defined by ventilator modes?

TL;DR I don't think APRV makes sense as a subclass of CPAP.  Maybe I'm being pedantic but if pressure changes during ventilation it is no longer continuous, and if it drops to 0 it's no longer positive.  That said, APRV is commonly called CPAP with release, so it's likely clinicians are not as overly concerned about this distinction.

**Comment by LCCarmody** [comment]
I think the easiest thing is to keep the terms 'flat' then, until (1) logical definitions are introduced or (2) a respiratory therapist is available.  I do need terms like 'noninvastive ventilation' and 'invasive ventilation', but I won't bog down with child terms.   This is the current structure (or lack thereof).  @pnrobinson do you have any  thoughts? ›
![Screen Shot 2020-04-01 at 5 00 19 PM](https://user-images.githubusercontent.com/26067008/78186196-4d2de500-743a-11ea-8c5d-9cc8082713fd.png)

**Comment by kshefchek** [comment]
It also makes it confusing that APRV is trademarked by Draeger, so on some vents this is called Bilevel, which shouldn't be confused bipap (which is mostly used as a non-invasive mode of ventilation)

**Comment by kshefchek** [comment]
My suggestion would be to change "positive pressure respiration" to "pressure controlled ventilation" (https://en.wikipedia.org/wiki/Pressure_control), and as a sibling we should have volume controlled ventilation.  APRV (or Bilevel) would be a subclass of pressure controlled ventilation.

**Comment by LCCarmody** [comment]
Okay, this does seem to be the [current convention](https://pubmed.ncbi.nlm.nih.gov/28918201/?from_term=pressure+controlled+ventilation&from_pos=1), so I will change it.  I will retain other terms as synonyms, since it does seem they are used (but possibly dated).
